industrial_ci/util.sh

```diff
@@ -86,15 +86,14 @@ function _end_fold_script {
     color_wrap=${2:-32}
 
     if [ $exit_code -eq "1" ]; then color_wrap=31; fi  # Red color
-    if [ -z $TRAVIS_FOLD_NAME ]; then
+    if ! [ -z $TRAVIS_FOLD_NAME ]; then
         ici_time_end $color_wrap
     else
 	echo "Previous Travis fold name not found. It might be either successful termination of the script, or wrong call. Skipping 'ici_time_end' anyway."
     fi
 
-    if [ $exit_code -eq "1" ]; then trap - ERR; fi  # Reset signal handler since the shell is about to exit.
     if [ "$DEBUG_BASH" ] && [ "$DEBUG_BASH" == true ]; then set -x; fi
-    if [ $exit_code -ne "-1" ]; then exit $exit_code; fi
+    if [ $exit_code -ne "-1" ]; then ici_exit $exit_code; fi
```

Comment:
I don't have a usecase for now, but only because this file is "util" we better address all the possibility we can think of.
Moving `ici_exit` sounds good to me.